pkg/engine/fetchit.go

```diff
@@ -136,6 +141,29 @@ func (fc *FetchitConfig) populateFetchit(config *FetchitConfig) *Fetchit {
 			config.TargetConfigs = append(config.TargetConfigs, reload)
 		}
 	}
+
+	// Check for ssh file
+	if config.GitAuth != nil {
+		if err := os.Setenv("SSH_KNOWN_HOSTS", "/opt/mount/.ssh/known_hosts"); err != nil {
+			cobra.CheckErr(err)
+		}
+		// GitAuth exists in config
+		keyPath := defaultSSHKey
```

Comment:
I missed this b4, move Lines 147 - 151 below the 
`if config.GitAuth.SSH {` - that only needs to run if accessing git over SSH

Reply:
I just updated this, should be good now. Added a couple comments to better explain, too.

Reply:
oh nice, thanks